sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala

```diff
@@ -427,6 +427,13 @@ private[hive] case class MetastoreRelation
           .getOrElse(sqlContext.defaultSizeInBytes))
     }
   )
+  override def sameResult(plan: LogicalPlan): Boolean = {
+      plan match {
+           case mr: MetastoreRelation =>
```

Comment:
thanks, marmbrus   but i don't know how to make  PRs against master,can you tell me? thanks again for your attension